src/SolverAPI.jl

```diff
@@ -469,8 +470,20 @@ function nl_to_aff_or_quad(::Type{T}, f::MOI.ScalarNonlinearFunction) where {T<:
                 return MOI.Utilities.operate(*, T, args[1], args[1])
             end
         else
-            h = get(_quad_ops, f.head, nothing)
-            isnothing(h) || return MOI.Utilities.operate(h, T, args...)
+            if f.head == :+
```

Comment:
We should have a more general solution here. The `head` could be `:-` with 1000 terms.

Reply:
It appears that `operate(-, ...)` is only defined for two arguments if I am not mistaken.

Reply:
it seems to have a single argument here https://github.com/jump-dev/MathOptInterface.jl/blob/53ec2693c8280d0e3f53336142c8739298eef0e5/src/Utilities/operate.jl#L1174 for unary minus (e.g. `-5`)